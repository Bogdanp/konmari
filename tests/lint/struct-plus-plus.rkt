#lang racket/base

(require racket/string)

(provide
 user
 set-user-username
 set-user-password
 set-user-password-hash)

(struct++ user
  ([username non-empty-string?]
   [password-hash bytes?])
  #:transparent)
