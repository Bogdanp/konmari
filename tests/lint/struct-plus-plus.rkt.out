/Users/bogdan/sandbox/konmari/tests/lint/struct-plus-plus.rkt:8:2:error:identifier 'set-user-password' provided but not defined
